README.md

```diff
@@ -31,7 +31,7 @@ condacolab.check()
 
 The default `condacolab.install()` provides Mambaforge, but there are other `conda` distributions to choose from:
 
-- `install_anaconda()`: This will install the Anaconda 2020.02 distribution, the last version that was built for Python 3.7. This contains [plenty of data science packages](https://docs.anaconda.com/anaconda/packages/old-pkg-lists/2020.02/py3.7_linux-64/), but they might be outdated by now.
+- `install_anaconda()`: This will install the Anaconda 2021.11 distribution, the last version that was built for Python 3.7. This contains [plenty of data science packages](https://docs.anaconda.com/anaconda/packages/old-pkg-lists/2021.11/py3.7_linux-64/), but they might be outdated by now.
 - `install_miniconda()`: This will install the Miniconda 4.9.2 distribution, using a version built for Python 3.7. Unlike Anaconda, this distribution only contains `python` and `conda`.
```

Comment:
```suggestion
- `install_miniconda()`: This will install the Miniconda 4.12.0 distribution, using a version built for Python 3.7. Unlike Anaconda, this distribution only contains `python` and `conda`.
```

Reply:
Can we also use this PR to update to Miniconda 4.12?

Link: https://repo.anaconda.com/miniconda/Miniconda3-py37_4.12.0-Linux-x86_64.sh

We also need to update the URL in `condacolab.py`, line 247.

Reply:
Done.